pkg/kv/kvserver/raftlog/encoding.go

```diff
@@ -78,15 +79,17 @@ const (
 	EntryEncodingSideloadedPrefixByte = byte(1)
 )
 
-// EncodeRaftCommand encodes a raft command (including the versioning prefix).
+// EncodeRaftCommand encodes a raft command of type EntryEncodingStandard or
+// EntryEncodingSideloaded.
 func EncodeRaftCommand(prefixByte byte, commandID kvserverbase.CmdIDKey, command []byte) []byte {
 	b := make([]byte, RaftCommandPrefixLen+len(command))
 	EncodeRaftCommandPrefix(b[:RaftCommandPrefixLen], prefixByte, commandID)
 	copy(b[RaftCommandPrefixLen:], command)
 	return b
 }
 
-// EncodeRaftCommandPrefix encodes the versioning prefix for a Raft command.
+// EncodeRaftCommandPrefix encodes the prefix for a Raft command of type
```

Comment:
Maybe "Header" would be a better name than "Prefix". Up to you.

Reply:
I'll leave as is.